Gemfile.lock

```diff
@@ -321,7 +321,7 @@ GEM
       activesupport (>= 4.2.0)
       rack (>= 1.1)
       rubocop (>= 1.7.0, < 2.0)
-    ruby-kafka (1.3.0)
+    ruby-kafka (1.4.0)
```

Comment:
* Refresh a stale cluster's metadata if necessary on Kafka::Client#deliver_message (#901).
* Fix Kafka::TransactionManager#send_offsets_to_txn (#866).
* Add support for murmur2 based partitioning.
* Add resolve_seed_brokers option to support seed brokers' hostname with multiple addresses (#877).
* Handle SyncGroup responses with a non-zero error and no assignments (#896).
* Add support for non-identical topic subscriptions within the same consumer group (#525 / #764).